armedia.repo

```diff
@@ -0,0 +1,6 @@
+[armedia]
```

Comment:
I guess, to continue the ArkCase branding vs Armedia branding, we want `arkcase.repo` and `[ArkCase]` and what not.

Reply:
Done!